packages/app/src/components/Randomizer/Randomizer.js

```diff
@@ -137,6 +146,8 @@ const Randomizer = ({
         {filter.visible ? 'Hide filter ' : 'Set filter '}
         {filter.visible ? <ExpandLess /> : <ExpandMore />}
       </button>
+      {restaurant && <p>Approval% : {(restaurant.resultAmount - restaurant.notSelectedAmount) / restaurant.resultAmount * 100}  </p>}
```

Comment:
"Approval% : XX" is not very informative. For better UX, the label could be changed to something more descriptive and maybe a tooltip should be added with some more detailed explanation what this number signifies.

Reply:
Fixed, added tooltip.